acs5countypoverty.py

```diff
@@ -0,0 +1,57 @@
+from census import Census
+from api_keys import census_key
+import json
+import requests
+import pandas as pd
+
+c = Census(census_key)
+
+#Estimate!!Total!!Population for whom poverty status is determined
+pop_table = 'S1701_C01_001E'
+#Estimate!!Below poverty level!!Population for whom poverty status is determined
+poverty_table = 'S1701_C02_001E'
+
+subject_table = 'https://api.census.gov/data/2018/acs/acs5/subject?'
+get = f'NAME,{pop_table},{poverty_table}'
+#county level request for IL (state 17)
+geo = 'county:*&in=state:17'
+url = f'{subject_table}get={get}&for={geo}&key={census_key}'
+response = requests.get(url)
+response
+
+
+#poverty_data is list of lists, where list elements are rows
+#header row: name, population total, population poverty, fip state, fip county
+poverty_data = response.json()
+poverty_data[:5]
+
+
+#Json format
+#{'countyfip': (countyfip = concatenation state and county fip numbers)
+    #{'metric_one': 1234,
+    #'metric_two': 5678}}
+    
+final_json = {}
+
+for d in poverty_data[1:]:
+    #set variables to list elements
+    name, pop_total_str, pop_poverty_str, fip_state, fip_county = d
+    #convert strings to ints
+    pop_total_int = int(pop_total_str)
+    pop_poverty_int = int(pop_poverty_str)
+    #concat strings
+    fip_final = fip_state + fip_county
+    #calculate percent poverty
+    pct_poverty = pop_poverty_int / pop_total_int * 100
+    #create county json
+    county_json = {'name_county':name, 'population_total': pop_total_int, 'population_poverty': pop_poverty_int, 'percent_poverty': pct_poverty}
+    #set county key to county json value
+    final_json[fip_final] = county_json
+
+
+#save file
+with open('acs5countypoverty_output.json', 'w') as f:
+    json.dump(final_json, f)
```

Comment:
Let's create a folder called `final_jsons` or something so that we can all write to the same location?

Reply:
Eventually, I think it'd be great to have Jenkins (or similar) run the script and then push the output up to our firebase database (or similar) when you merge the PR. But for now, I think this makes sense.

Reply:
Question: Since the script produces these jsons, I think maybe we shouldn't store them in the repo at all? It creates the opportunity for the script and the output json to get out of sync. Instead, maybe we should just document really well how to run the script to get the json? 

Thoughts?

Reply:
I think we can have the file on the repo for now so FE can reference, until we build in the documentation.